### DISCUSSION TITLE
Working Configuration example for PMD with Sarif-Output-Format

### DISCUSSION THREAD

**Discussion by hmmh-sven-scheil** [pmd/pmd#5168] [Q&A] [answered]
I'm using pmd with gradle-plugin and want to generate a sarif report, which should finally be uploaded to github codescan analysis.

I can't find any working example for it. Is this because it does not work at all?

**Comment by jsotuyod** [comment, marked as answer]
The PMD Gradle plugin is fully managed by Gradle. They only support html and xml reports (see https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/quality/PmdReports.html ) or text output to stdout.

My recommendation would be to bring the issue to them, but I'd guess they would love a PR, are very unlikely to prioritize this themselves.

**Reply by hmmh-sven-scheil** [reply]
Thanks for your quick reply!
